examples/model_config.toml

```diff
@@ -36,17 +36,44 @@
     log_file_level = "DETAIL"
     log_on_error_file_path = "log_error.txt"
     notify_slack = false
-    use_emme_logbook = true
+    use_emme_logbook = false
```

Comment:
@xyzjayne why not using emme logbook?

Reply:
WIll change it back. Thanks for spotting this.